packages/terra-table/docs/guides/SortedTable.md

```diff
@@ -0,0 +1,163 @@
+```jsx
+  createHeaderCell(key, title) {
+    return (
+      <HeaderCell
+        key={key}
+        metaData={{ key }}
+        onSelect={this.handleSortClick}
+      >
+        {title}
+      </HeaderCell>
+    );
+  }
+```
+For the ability to toggle sorting we set "isSelectable" for all header cells.
+```jsx
+  createHeaderCell(key, title) {
+    return (
+      <HeaderCell
+        key={key}
+        isSelectable
+        metaData={{ key }}
+        onSelect={this.handleSortClick}
+      >
+        {title}
+      </HeaderCell>
+    );
+  }
+```
+Finally we need to check if the header cell matches the sortColumn.key in state. In the case the header cell isn't sorted no sort indicator value should be passed.
+```jsx
+  createHeaderCell(key, title) {
+    let sort;
+    if (this.state.sortColumn && this.state.sortColumn.key === key) {
+      sort = this.state.sortColumn.direction;
+    }
+  
+    return (
+      <HeaderCell
+        key={key}
+        isSelectable
+        sort={sort}
+        metaData={{ key }}
+        onSelect={this.handleSortClick}
+      >
+        {title}
+      </HeaderCell>
+    );
+  }
+```
+In this example a simple object sort is parses the data, followed by a check for reversal.
+```jsx
+const sortData = (data, sortColumn) => {
+  if (!sortColumn) {
+    return data;
+  }
+
+  const dataCopy = Object.assign([], data);
+  dataCopy.sort((a, b) => {
+    const x = a.cells[columns.indexOf(sortColumn.key)].title.toLowerCase();
+    const y = b.cells[columns.indexOf(sortColumn.key)].title.toLowerCase();
+    if (x < y) { return -1; }
+    if (x > y) { return 1; }
+    return 0;
+  });
+
+  return sortColumn.direction === 'asc' ? dataCopy : dataCopy.reverse();
+};
+```
+Next we fill in the static methods for the example table rows.
+```jsx
+const columns = ['column-0', 'column-1', 'column-2'];
+
+const createCell = cell => (
+  <Cell key={cell.key}>
+    <Placeholder title={cell.title} style={{ height: '50px', padding: '0' }} />
+  </Cell>
+);
+
+const createCellsForRow = cells => cells.map(cell => createCell(cell));
+
+const createTableRow = itemData => <Row key={itemData.key}>{createCellsForRow(itemData.cells)}</Row>;
+```
+Then we can implement a method to loop through our data and create the table row with our methods and call it from our render method. 
+```jsx
+  createTableRows(data) {
+    const sortedData = sortData(data, this.state.sortColumn);
+    return sortedData.map(childItem => createTableRow(childItem));
+  }
+
+  render() {
+    return (
+      <Table
+        header={(
+          <Header>
+            {this.createHeaderCell('column-0', 'Breakfast')}
+            {this.createHeaderCell('column-1', 'Animals')}
+            {this.createHeaderCell('column-2', 'Flatware')}
+          </Header>
+        )}
+      >
+        {this.createTableRows(mockData)}
+      </Table>
+    );
+  }
+}
+  ```
+  Using these steps we get the following example:
```

Comment:
Oops looks like the ex is missign from the readme view

Reply:
The example is built with the the doc template.